pkg/internal/checkup/executor/testpmd/console.go

```diff
@@ -67,15 +67,15 @@ const (
 
 const testpmdPrompt = "testpmd> "
 
-func NewTestpmdConsole(vmiSerialClient vmiSerialConsoleClient, namespace, vmiEastNICPCIAddress,
-	vmiEastEthPeerMACAddress, vmiWestNICPCIAddress, vmiWestEthPeerMACAddress string, verbosePrintsEnabled bool) *TestpmdConsole {
+func NewTestpmdConsole(vmiSerialClient vmiSerialConsoleClient, namespace, vmiUnderTestEastNICPCIAddress,
+	trafficGenEastMACAddress, vmiUnderTestWestNICPCIAddress, trafficGenWestMACAddress string, verbosePrintsEnabled bool) *TestpmdConsole {
 	return &TestpmdConsole{
 		vmiSerialClient:          vmiSerialClient,
 		namespace:                namespace,
-		vmiEastEthPeerMACAddress: vmiEastEthPeerMACAddress,
-		vmiWestEthPeerMACAddress: vmiWestEthPeerMACAddress,
-		vmiEastNICPCIAddress:     vmiEastNICPCIAddress,
-		vmiWestNICPCIAddress:     vmiWestNICPCIAddress,
+		vmiEastEthPeerMACAddress: trafficGenEastMACAddress,
+		vmiWestEthPeerMACAddress: trafficGenWestMACAddress,
```

Comment:
Why do fields who's name contains `EthPeer` are kept here, while they were changed in executor?

Reply:
because in the testpmd context, the traffic-gen is the Ethernet peer, but outside of that context, there is no specific meaning to Ethernet peer - both are each other's peers..